report/src/main/webapp/assets/sass/_sidebar.scss

```diff
@@ -213,6 +207,15 @@
       }
     }
   }
+
+  .fontawesome {
```

Comment:
Seems like styles for fontawesome class are present in few of scss files, is it possible to move them all to one file? Or at least some of them that aren't nested in some file-specific classes?